pkg/bpf/map_linux.go

```diff
@@ -115,7 +115,7 @@ type Map struct {
 	NonPersistent bool
 
 	// DumpParser is a function for parsing keys and values from BPF maps
-	dumpParser DumpParser
+	DumpParser DumpParser
```

Comment:
Does this need to be public? I don't see any external use of this field.

Reply:
We need this in `bpf_lb_list_maglev.go`. Please see a comment there explaining why it's needed.